hikari/api/interaction_server.py

```diff
@@ -132,20 +132,6 @@ async def on_interaction(self, body: bytes, signature: bytes, timestamp: bytes)
             the interaction request.
         """
 
-    @typing.overload
-    @abc.abstractmethod
-    def get_listener(
-        self, interaction_type: typing.Type[command_interactions.CommandInteraction], /
-    ) -> typing.Optional[ListenerT[command_interactions.CommandInteraction, _MessageResponseBuilderT]]:
-        ...
-
-    @typing.overload
-    @abc.abstractmethod
```

Comment:
You probably want to add another case to these overloads instead of just removing them

Reply:
Any reason for doing that? As far as I can see there's no point in keeping the overloads.

Reply:
for more concise typing